api/paidAction/receive.js

```diff
@@ -0,0 +1,70 @@
+import { PAID_ACTION_PAYMENT_METHODS } from '@/lib/constants'
+import { toPositiveBigInt } from '@/lib/validate'
+import { notifyDeposit } from '@/lib/webPush'
+import { numWithUnits, msatsToSats } from '@/lib/format'
+
+export const anonable = false
+
+export const paymentMethods = [
+  PAID_ACTION_PAYMENT_METHODS.P2P,
+  PAID_ACTION_PAYMENT_METHODS.OPTIMISTIC
```

Comment:
Because only logged in stackers can receive, this action doesn't create new db records, and hold invoices should be avoided unless we want to do refunds, we don't allow pessimistic payment flows.